rxbilling/src/main/java/com/gen/rxbilling/client/RxBilling.kt

```diff
@@ -11,41 +11,39 @@ import io.reactivex.Flowable
 import io.reactivex.Single
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.subjects.PublishSubject
-import timber.log.Timber
 
 interface RxBilling : Connectable<BillingClient> {
 
     override fun connect(): Flowable<BillingClient>
 
     fun observeUpdates(): Flowable<PurchasesUpdate>
 
-    fun getPurchases(): Single<List<Purchase>>
+    fun getPurchases(skuType: String): Single<List<Purchase>>
 
-    fun getSubscriptions(): Single<List<Purchase>>
+    fun getPurchaseHistory(skuType: String): Single<List<PurchaseHistoryRecord>>
```

Comment:
can you please add a StringDef anotation to the argument to narrow down possible input for `getPurchaseHistory` and `getPurchases`?
`fun getPurchaseHistory(@BillingClient.SkuType skuType: String): Single<List<PurchaseHistoryRecord>>`